File: Android/JavaTarefas/Tarefa1Classes/Retangulo.java
package Tarefa1Classes;

public class Retangulo {
    protected int comprimento;
    protected int largura;

    public void setComprimento(int comprimento) {
        this.comprimento = comprimento;
    }

    public int getComprimento() {
        return comprimento;
    }

    public void setLargura(int largura) {
        this.largura = largura;
    }

    public int getLargura() {
        return largura;
    }

    public int getPisos() {
        return comprimento * largura;
    }

    public int getRodape() {
        return 2 * (comprimento + largura);
    }
}

File: Android/Kotlin/assessores/src/main/kotlin/Retangulo.kt
class Retangulo(val altura: Int, val largura: Int) {
    val isQuadrado: Boolean
        get(){
            return altura == largura
        }
}

fun main(){
    val retangulo: Retangulo = Retangulo(10, 10)
    println(retangulo.isQuadrado)
}